src/assets/back.vue

```diff
@@ -0,0 +1,6 @@
+<template>
```

Comment:
This could be reduced to this

`<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 100 100"><path fill="#958A8A" d="M79.167 20.834H40.958a12.5 12.5 0 0 0-8.833 3.667L9.542 47.042a4.167 4.167 0 0 0 0 5.917l22.583 22.542a12.5 12.5 0 0 0 8.833 3.666h38.209a12.5 12.5 0 0 0 12.5-12.5V33.334a12.5 12.5 0 0 0-12.5-12.5Zm4.166 45.833a4.167 4.167 0 0 1-4.166 4.167H40.958A4.376 4.376 0 0 1 38 69.626L18.375 50 38 30.376a4.375 4.375 0 0 1 2.958-1.209h38.209a4.167 4.167 0 0 1 4.166 4.167v33.333ZM69.625 38.71a4.168 4.168 0 0 0-5.917 0l-5.375 5.417-5.375-5.417a4.184 4.184 0 0 0-5.916 5.917L52.458 50l-5.416 5.375a4.165 4.165 0 0 0 0 5.916 4.167 4.167 0 0 0 5.916 0l5.375-5.416 5.375 5.416a4.166 4.166 0 0 0 5.917 0 4.167 4.167 0 0 0 0-5.916L64.208 50l5.417-5.375a4.167 4.167 0 0 0 0-5.917Z"/></svg>`